homeassistant/components/zha/core/channels/__init__.py

```diff
@@ -242,6 +254,8 @@ def add_all_channels(self) -> None:
                     # on power configuration channel per device
                     continue
                 self._channels.power_configuration_ch = channel
+            elif channel.name == const.CHANNEL_IDENTIFY:
+                self._channels.identify_ch = channel
```

Comment:
I haven't checked if there're devices with multiple "identify" clusters, but in case of multiple endpoints and identify clusters, this would keep only the last identify cluster. Is it the intended behavior?

Reply:
what device you've been testing with? Does it have identify cluster on the same endpoint is "on_off" cluster?

Reply:
> I haven't checked if there're devices with multiple "identify" clusters, but in case of multiple endpoints and identify clusters, this would keep only the last identify cluster. Is it the intended behavior?

It should keep first cluster because of setter guard here: https://github.com/home-assistant/home-assistant/pull/32234/files#diff-accbacd0a97d6988089b194a418cdff5R72. Similar to power config channel.
There are devices like device_no=5 in zha_devices_list.py with multiple identify clusters, but I didn't find any light.

> what device you've been testing with? Does it have identify cluster on the same endpoint is "on_off" cluster?

Yes, I have Tradfri bulb similar to device_no=17

Reply:
I think we should use the `identify` cluster from the same endpoint as the `on_off` cluster. There are multi-gang wall light switches, that control multiple lights and each endpoint has its own identify cluster. Take a look at `device_no: 92` or `device_no: 41` -- aqara multi channel relay.
Just add `CHANNEL_IDENTIFY` to the `aux_channels` and and use it for commands. Would make it work with two more additional devices ;)